docs/configuration/cluster-misconfigurations.rst

```diff
@@ -0,0 +1,118 @@
+Kubernetes Misconfigurations (Popeye)
+************************************************
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes clusters and reports potential issues with resources and configurations.
+By default, every instance of Robusta that's connected to the UI will run a Popeye scan on startup. Further Popeye scans can be triggered in the UI, and all scans can be viewed there.
+
+With or without the UI, you can configure additional scans on a :ref:`schedule <Scheduled>` as shown below.
+
+.. code-block:: yaml
+    :name: cb-popeye-set-periodic-scan
+
+    customPlaybooks:
+    - triggers:
+      - on_schedule:
+          fixed_delay_repeat:
+            repeat: 1 # number of times to run or -1 to run forever
+            seconds_delay: 604800 # 1 week
+      actions:
+      - popeye_scan:
+          spinach: |
+            popeye:
+              excludes:
+                  v1/pods:
+                  - name: rx:kube-system
+      sinks:
+      - "robusta_ui_sink"
+
+The results can be sent as a PDF to Slack or to the Robusta UI.
+
+.. grid:: 1 1 1 1
+
+    .. grid-item::
+
+        .. md-tab-set::
+
+            .. md-tab-item:: Slack
+
+                .. image:: /images/popeye_slack_example.png
+                    :width: 1000px
+
+            .. md-tab-item:: Robusta UI
+
+                .. image:: /images/popeye_example.png
+                    :width: 1000px
+
+
+.. Note::
+
+    Other sinks like MSTeams are not supported yet.
+
+.. robusta-action:: playbooks.robusta_playbooks.popeye.popeye_scan on_schedule
+
+    You can trigger a Popeye scan at any time, by running the following command:
+
+    .. code-block:: bash
+
+        robusta playbooks trigger popeye_scan
+
+Taints, Tolerations and NodeSelectors
```

Comment:
Can you align this a little more around jobs-to-be-done?
E.g. "Running Popeye in ARM Clusters or On Specific Nodes" and update the content slightly. (First give motivation then go into the how.)